spec.bs

```diff
@@ -612,8 +614,6 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       |signalsUrl|, |keys|, |igNames|, |buyerExperimentGroupId|.
     1. Let |trustedBiddingSignals| be the result of [=fetching resource=] with |fullSignalsUrl|,
       "application/json", and null.
-    1. TODO: If |trustedBiddingSignals| is failure, fail the auction with manually_aborted set to

```

Comment:
I think we still need to handle the fetch failure, even if we don't fail the auction -- I'll try to address this in https://github.com/WICG/turtledove/pull/533.